addons/hr/views/hr_employee_public_views.xml

```diff
@@ -134,46 +134,52 @@
                     <templates>
                         <t t-name="kanban-box">
                         <div class="oe_kanban_global_click o_kanban_record_has_image_fill o_hr_kanban_record">
-                            <field name="image_1024" class="o_kanban_image_fill_left d-block" preview_image="image_128"
+                            <t t-if="!record.image_1024.raw_value">
+                                <field name="avatar_128" class="o_kanban_image_fill_left d-block"
                                 widget="background_image" options="{'zoom': true, 'zoom_delay': 1000}"/>
-
+                            </t>
+                            <t t-else="">
+                                <field name="image_1024" class="o_kanban_image_fill_left d-block" preview_image="image_128"
+                                    widget="background_image" options="{'zoom': true, 'zoom_delay': 1000}"/>
+                            </t>
                             <div class="oe_kanban_details">
                                 <div class="o_kanban_record_top">
                                     <div class="o_kanban_record_headings">
                                         <strong class="o_kanban_record_title">
+                                            <div class="float-end">
+                                                <t t-if="record.show_hr_icon_display.raw_value">
+                                                    <div class="o_employee_availability d-flex flex-row align-items-start">
```

Comment:
I'd remove the margins on the sides for `o_employee_availability`, the distance from the right border feels weird